src/dvclive/live.py

```diff
@@ -87,6 +95,23 @@ def __init__(
         else:
             self._cleanup()
 
+        self._latest_studio_step = self.get_step()
+
+        if self._report == "studio":
+            from scmrepo.git import Git
```

Comment:
Do we need to add scmrepo to setup.cfg as an extra?

Reply:
Would probably be the right thing to do, although I don't think it matters because the feature can only work if `dvc` is installed.